### PR TITLE
[HDInsight] Add warning message for break changes which plan to release in Nov.

### PR DESCRIPTION
**Pull request by yuc-Li**
## Description
Add warning message for break changes in this [PR](https://github.com/Azure/azure-powershell/pull/22428) which plan to release in Nov. 

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by aim-for-better** [comment]
Please check if you need to update the changelog.md and submit this PR.

File: src/HDInsight/HDInsight/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,11 @@
 -->
 ## Upcoming Release
 
+## Version 6.1.0
+* Added warning message for planning to replace the type of property `DiskEncryption` of type `Microsoft.Azure.Commands.HDInsight.Models.AzureHDInsightCluster` from `Microsoft.Azure.Management.HDInsight.Models.DiskEncryptionProperties` to `Azure.ResourceManager.HDInsight.Models.HDInsightDiskEncryptionProperties`.
```
Comment (aim-for-better) [inline src/HDInsight/HDInsight/ChangeLog.md:23]: I suggest to add one more sentence here like "This change adds some warning messages for the incoming breaking change, the details are as below:".   It is like summary then concrete items

File: src/HDInsight/HDInsight/ChangeLog.md [hunk 2]
```diff
@@ -19,6 +19,12 @@
 -->
 ## Upcoming Release
 
+## Version 6.1.0
+* This change adds some warning messages to the incoming break changes for the next version, with detailed information as follows:
```
Comment (YanaXu) [inline src/HDInsight/HDInsight/ChangeLog.md:23]: Please move the 3 new lines to "Upcoming Release".

**Review by YanaXu — APPROVED** [review]

**Pull request merged**